tools/CleanOutput.py

```diff
@@ -0,0 +1,77 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import os
+import re
```

Comment:
Is there a reason not to alphabetize these?

Reply:
it's not part of pep8 to sort imports. we should do it anyway